src/components/SummaryBox/SummaryBoxHeading/SummaryBoxHeading.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactNode } from 'react'
+import classnames from 'classnames'
+
+interface SummaryBoxHeadingProps {
+  children: ReactNode
+  className?: string
+  headingLevel: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6'
+}
+
+export const SummaryBoxHeading = ({
+  children,
+  className,
+  headingLevel,
+  ...h3Props
+}: SummaryBoxHeadingProps &
+  JSX.IntrinsicElements['h3']): React.ReactElement => {
+  const classes = classnames('usa-summary-box__heading', className)
+  const Heading = headingLevel
+  return (
+    <Heading className={classes} {...h3Props} data-testid="summaryBoxHeading">
```

Comment:
nit: I prefer to save the data-testid for when something really can't easily be queried from the dom by conventional means.

Before this PR the test used the following: `expect(getByRole('heading')).toBeInTheDocument()`. I think it could also work with `getByText`, knowing what the heading's content is.

Reply:
ah, I forgot about getByRole!